docs/tasks/configure-pod-container/two-container-pod.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: two-containers
+spec:
+
+  volumes:
+  - name: shared-data
+    emptyDir: {}
+
+  containers:
+
+  - name: nginx-container
+    image: nginx
+    volumeMounts:
+    - name: shared-data
+      mountPath: /pod-data
```

Comment:
I think we could even mount this to the directory nginx serves from and write the index.html file there.  WDYT?